.build-cgo

```diff
@@ -0,0 +1 @@
+// #cgo CFLAGS: -DSQLITE_ENABLE_JSON1
```

Comment:
Wow, how did you figure this out :D That's great!

Reply:
It shouldn't have be there it was a memo. 
I think we do not need it for it to works